src/main/java/com/iota/iri/service/dto/GetNodeInfoResponse.java

```diff
@@ -56,70 +61,160 @@ public static AbstractResponse create(String appName, String appVersion, int jre
 		return res;
 	}
 
+    /**
+     * Gets the app name
+     *
+     * @return The app name.
+     */
```

Comment:
hmm
All of this is used by the mdx generator?
If no, then this javadoc is completely extraneous and you can delete it.

If yes, then must you violate DRY?
won't it suffice to just write `@return The app name`.

Reply:
Either text or @return is needed. If we need a long explanation, putting it in the @return isnt really good practise. (Example: AttachToTangleResponse.getTrytes)

If we remove the @return, some javadoc tools might stop working(right?), since this is how it should be done. Want me to remove the@retuns from public getters?